src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -605,6 +605,11 @@ private void InitializePackagesFolderInfo(string packagesFolderPath, bool exclud
                 IEnumerable<SourceRepository> secondarySources,
                 CancellationToken token)
         {
+            if (packageIdentities == null)
```

Comment:
Given that this is a private method, can a public method run into this?

Reply:
https://github.com/NuGet/NuGet.Client/blob/74b387a2daee9ec136d3e50e41f7e3586459ad26/src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs#L578

Reply:
thanks.